### ISSUE TITLE
[PEOPLE] Figure out data structure for bio

### ISSUE THREAD

**Issue by gideonthomas** [mozilla/network-api#38]
Currently, the front-end displays bio as a list. This is a bit weird on the back-end as it suggests making "bio" a complex data structure, the implication being figuring out how to show this via the admin interface in a way that is not confusing.

Since this is the first "text-rich" field that we have encountered, let's have a decision on what type of data structure this should be.

#### Options
- Bullet points (like in the comps)
   - We will need to fix the number of bullet points in this case (we can't change this later on)
   - We need a max character length
- Markdown
   - This is more free form. We can probably show a preview of the rendered markdown in the admin view
   - Don't know what the repercussions are to design
   - We can set a max char count if it makes sense
   - Front-end would need to parse this markdown into markup

It would be nice to have a decision on this soon so that we can finish off the people app without being blocked on this.

cc @xmatthewx, @simonwex, @gvn, @alanmoo, @cadecairos

**Comment by cadecairos** [comment]
My preference here would be a limited number of text fields. Markdown is added complexity that I think we should avoid, and I will never agree to storing full on HTML in the field so don't ask :)

**Comment by gideonthomas** [comment]
I think we're going for a single textfield if I remember correctly from yesterday's call.

**Comment by gvn** [comment]
Yes, single text field is approved.
